grpc/src/main/java/com/linecorp/armeria/server/grpc/GrpcServiceBuilder.java

```diff
@@ -443,6 +447,20 @@ public GrpcServiceBuilder unframedGrpcErrorHandler(UnframedGrpcErrorHandler unfr
         return this;
     }
 
+    /**
+     * Set a custom HTTP status code mapper. This is useful to serve custom HTTP status code when using unframed
+     * gRPC service.
+     * @param unframedGrpcStatusFunction The function which maps the {@link Throwable} or grp {@link Status}
+     *                                   code to an {@link HttpStatus} code.
+     */
+    @UnstableApi
+    public GrpcServiceBuilder unframedGrpcStatusMapping(
```

Comment:
`unframedGrpcStatusMapper`?
and for the class name as well?

Reply:
Do you mean to rename `UnframedGrpcStatusFunction` to `UnframedGrpcStatusMapper` as well?
Actually, I followed the naming of [`exceptionMapping()`](https://github.com/line/armeria/blob/master/grpc/src/main/java/com/linecorp/armeria/server/grpc/GrpcServiceBuilder.java#L570) similarly :)

Reply:
Then, probably better to rename it to `UnframedGrpcStatusMappingFunction`?

Reply:
SGTM!

Reply:
Renamed